src/components/NotesList/Note/Note.jsx

```diff
@@ -0,0 +1,102 @@
+import PropTypes from "prop-types";
+
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import CloseIcon from "@material-ui/icons/Close";
+import EditIcon from "@material-ui/icons/Edit";
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import CardHeader from "@material-ui/core/CardHeader";
+
+import { CutDescription, DateFormatter } from "utils/utils.js";
+
+import useStyles from "./styled";
+
+const Note = ({
+  note,
+  activeNoteId,
+  setActiveNoteId,
+  isEditMode,
+  setIsEditMode,
+  isSharedMode
+}) => {
+  const classes = useStyles();
+  const onCloseNote = (event) => {
+    event.stopPropagation();
+    setActiveNoteId(-1);
+  };
+  const onEditNote = (event) => {
+    event.stopPropagation();
+    setActiveNoteId(note.id);
+    setIsEditMode(true);
+  };
+  const onShowNote = (noteId) => {
+    !isSharedMode && setIsEditMode(false);
+    setActiveNoteId(noteId);
+  };
+  return (
+    <Card
+      className={[
+        classes.root,
+        note.id === activeNoteId ? classes.active : "",
+      ].join(" ")}
```

Comment:
Can you please move this expression at the top of component to avoid adding calculations inside the tags.